pytest_plugins/requirements/update_requirements.py

```diff
@@ -45,3 +47,5 @@ def pytest_report_header(config):
         if config.getoption('update_all_reqs'):
             print('Updating the optional requirements on demand ....')
             updater.install_opt_deviations()
+    else:
+        print(f'{Colored.GREEN}Optional Requirements are up to date!{Colored.RESET}')
```

Comment:
```suggestion
        print(f'{Colored.GREEN}Optional Requirements are up to date.{Colored.RESET}')
```

Reply:
Not a blocker comment ! Ignoring for now !